R/readme.R

```diff
@@ -33,7 +33,7 @@ use_readme_rmd <- function(base_path = ".", open = TRUE) {
     base_path = base_path
   )
 
-  if (uses_git(base_path) && !file.exists(base_path, ".git", "hooks", "pre-commit")) {
```

Comment:
`use_git_hook()` warns if the file exists and is different from that being created